doc/radosgw/s3-notification-compatibility.rst

```diff
@@ -97,14 +97,30 @@ Event Types
 +--------------------------------------------------------+-----------------------------------------+
 | ``s3:ObjectLifecycle:Transition:NonCurrent``           | Ceph extension                          |
 +--------------------------------------------------------+-----------------------------------------+
+| ``s3:LifecycleExpiration:*``                           | Supported                               |
++--------------------------------------------------------+-----------------------------------------+
+| ``s3:LifecycleExpiration:Delete``                      | Supported                               |
++--------------------------------------------------------+-----------------------------------------+
+| ``s3:LifecycleExpiration:DeleteMarkerCreated``         | Supported                               |
++--------------------------------------------------------+-----------------------------------------+
+| ``s3:LifecycleTransition``                             | Supported                               |
++--------------------------------------------------------+-----------------------------------------+
 | ``s3:ObjectSynced:*``                                  | Ceph extension                          |
 +--------------------------------------------------------+-----------------------------------------+
 | ``s3:ObjectSynced:Create``                             | Ceph Extension                          |
 +--------------------------------------------------------+-----------------------------------------+
-| ``s3:ObjectSynced:Delete``                             | Defined, Ceph extension (not generated) |
+| ``s3:ObjectSynced:Delete``                             | Ceph extension                          |
```

Comment:
should we document the fact that the S3 standard events are the recommended ones? (and long term the duplicate ceph extension notifications will be removed?)

Reply:
I think deprecating things is jumping the gun